akka-actor-tests/src/test/scala/akka/actor/ExtensionSpec.scala

```diff
@@ -6,7 +6,7 @@ package akka.actor
 import java.util.concurrent.atomic.AtomicInteger
 
 import akka.testkit.EventFilter
-import akka.testkit.TestKit._
+import akka.testkit.scaladsl.TestKit._
```

Comment:
not sure it is worth renaming the scala api, breaking a lot of test code for users without much value